### PR TITLE
Fix build on EL7

### PR DESCRIPTION
**Pull request by brianjmurrell**
**Changes**
Add `/usr/local/bin` to `$PATH` as that is where the MS build of _dotnet_ puts `dotnet`.

Update `fedora/Makefile` with enhancements from https://github.com/jellyfin/jellyfin-web/commit/92cb985951de3227465ac723c4b92b2abdf49abd

### CODE REVIEW

**Comment by brianjmurrell** [comment]
Builds on all RH platforms now:
image

**Review by crobibero — APPROVED** [review]

**Comment by brianjmurrell** [comment]
Can I get a second review on this and landed if approved?  Hoping to see this land before alpha4 so that we can start to get useful packages for people to test with.

@joshuaboniface perhaps?

**Review by joshuaboniface — APPROVED** [review]

**Comment by brianjmurrell** [comment]
Awesome!  Bring on alpha4!  :-)

**Pull request merged**
